configure.ac

```diff
@@ -277,7 +301,7 @@ AC_LINK_IFELSE([AC_LANG_PROGRAM(
 SECTION_TITLE([RUNNING TESTS ON libzfs.h HEADER])
 
 
-if [[ "$zof" = false ]]; then
+if [[ "$zof" = false ]] || [[ "x$with_o3x" != "x" ]]; then
```

Comment:
With the current layout, when can `zof` be `false` with `with_o3x` being not empty while building on macos ?
I suspect that won't happen and I am not sure I follow the motivation behind this conditional then. Can you please elaborate ?

Reply:
unlike FreeBSD, macos build requires `sys/zfs_ioctl.h`. This conditional will change once I switch from `zof` to `o3x` variable.